code/game/objects/random/food.dm

```diff
@@ -2,23 +2,25 @@
 	name = "random preserved rations"
 	icon_state = "food-green"
 	tags_to_spawn = list(SPAWN_RATIONS)
+	spawn_nothing_percentage = 90
```

Comment:
You should not apply a no spawn chance to a spawner that does not specify that it is a low chance spawner.

Reply:
Option 2 is removing the spawn entirely, these rations specifically are a big problem since they come with no drawbacks whatsoever.

Reply:
There are many more options to do this.

Reply:
Namely?

Reply:
eliminate spawners with no chance to spawn anything on the map for example (or in the closets).

Reply:
Keeping this as resolved, beats replacing every instance of /obj/spawner/junkfood